src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -5884,4 +5973,47 @@ def test_aks_enable_addon_with_azurekeyvaultsecretsprovider(self, resource_group
         cmd = 'aks delete --resource-group={resource_group} --name={name} --yes --no-wait'
         self.cmd(cmd, checks=[
             self.is_empty(),
+        ])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus2')
+    @AKSCustomRoleBasedServicePrincipalPreparer()
+    def test_aks_update_labels(self, resource_group, resource_group_location, sp_name, sp_password):
```

Comment:
This test case will fail in live mode ([log](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=48698070&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=dc59ccd1-231f-538b-777f-33a592c7ca57&l=414)).